src/components/AccountLoginForm.jsx

```diff
@@ -120,14 +120,16 @@ const AccountLoginForm = props => {
 
   return (
     // We use a <div> instead of a <form> to disable the "use password for" function of Chrome
-    <div className={styles['account-form-login']}>
+    <div className={styles['account-form-login']} role="form">
       {/* Error */}
       {error && (
         <p className="errors">{t('account.message.error.bad_credentials')}</p>
       )}
 
       {/* Fields */}
-      {!!editableFields && editableFields.map(renderField)}
+      <fieldset className={styles['account-form-fieldset']}>
```

Comment:
Is it not a problem here if we don't have any fields (pure OAuth konnector)?
It could be better to check `{!!editableFields && editableFields.length && <fieldset....}` to render the `fieldset` and then inside it use `{editableFields.map(renderField)}`